test/account_test.rb

```diff
@@ -26,12 +26,34 @@ def app
   end
 
   describe "POST /account/accept/ok" do
-    it "redirects to the signup app" do
+    it "redirects to the signup app on success" do
       stub(Identity::Config).redirect_all_signups { true }
-      post "/account/accept/ok"
+      post "/account/accept/ok", token: 'foo'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.